databricks/koalas/datetime.py

```diff
@@ -0,0 +1,154 @@
+#
+# Copyright (C) 2019 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+Date/Time related functions on Koalas Series
+"""
+
+import functools
+from pyspark.sql.types import DateType, TimestampType, LongType, StringType
+
+from databricks.koalas.series import Series
+from databricks.koalas.utils import lazy_property
+
+import pyspark.sql.functions as F
+
+
+def defer_to_pandas(output_type):
```

Comment:
add type annotation for output_type?

Reply:
also these two functions are not specific to datetime right? perhaps we should put that in utils.py.

and I'd name it either `apply_pandas_function` and `apply_spark_function`. 

One thing is we should make sure the generated doc is still readable, i.e. it shows the parameters for the functions.

Reply:
you should try add this to the api reference in docs directory and see how they render

Reply:
Sounds good. I haven't been able to generate docs (missing usr/bin/xcrun). I will spend some time figuring it out.

Reply:
You might need to install the xcode command line tools on mac.